src/depends.py

```diff
@@ -58,35 +66,49 @@ def check_sqlite():
     support in python version for specifieed platform.
     """
     if sys.hexversion < 0x020500F0:
-        logger.error('The sqlite3 module is not included in this version of Python.')
+        logger.error(
+            'The sqlite3 module is not included in this version of Python.')
         if sys.platform.startswith('freebsd'):
-            logger.error('On FreeBSD, try running "pkg install py27-sqlite3" as root.')
+            logger.error(
+                'On FreeBSD, try running "pkg install py27-sqlite3" as root.')
         return False
     try:
```

Comment:
Can we place this in the starting?
 try:
         import sqlite3
     except ImportError:
         logger.error('The sqlite3 module is not available')
         return False

Reply:
return where? This returns from particular function.

Reply:
@g1itch , better to place this in a function and call it where it requires

Reply:
OK, I'll try.

Reply:
I figured out that this module duplicates the code of `checkdeps`.